src/omero_zarr/masks.py

```diff
@@ -303,30 +304,30 @@ def save(self, masks: List[omero.model.Shape], name: str) -> None:
             ignored_dimensions,
             check_overlaps=True,
         )
-        # For v0.3 ngff we want to reduce the number of dimensions to
+
+        metadata = marshal_axes(self.image, levels=1)
+
+        # For v0.3+ ngff we want to reduce the number of dimensions to
         # match the dims of the Image.
         dims_to_squeeze = []
-        axes = []
         for dim, size in enumerate(self.image_shape):
             if size == 1:
                 dims_to_squeeze.append(dim)
-            else:
-                axes.append("tczyx"[dim])
         labels = np.squeeze(labels, axis=tuple(dims_to_squeeze))
 
         scaler = Scaler(max_layer=input_pyramid_levels)
         label_pyramid = scaler.nearest(labels)
         pyramid_grp = out_labels.require_group(name)
 
         write_multiscale(
-            label_pyramid, pyramid_grp, axes=axes
+            label_pyramid, pyramid_grp, **metadata
         )  # TODO: dtype, chunks, overwite
 
         # Specify and store metadata
         image_label_colors: List[JSONDict] = []
         label_properties: List[JSONDict] = []
         image_label = {
-            "version": "0.3",
+            "version": "0.4",
```

Comment:
Use `VERSION` (or `CurrentFOrmat().version`)?

Reply:
Done in 9194fdd